config/manager/kustomization.yaml

```diff
@@ -1,2 +1,8 @@
 resources:
 - manager.yaml
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+images:
+- name: controller
+  newName: karthikkn27/capioperator
```

Comment:
I assume we want to change this before merging?

Reply:
Yes, removed it